codec-dns/src/main/java/io/netty/handler/codec/dns/DnsRecord.java

```diff
@@ -82,4 +82,9 @@
      * Returns the time to live after reading for this resource record.
      */
     long timeToLive();
+
+    /**
+     * Decodes the record data.
+     */
+    void decodeRdata();
```

Comment:
a Record should be just a "Pojo". Decoding does belong to a decoder and not the record

Reply:
The decode operation has been moved to the decoder :-). @normanmaurer would you please review it again ?